pkg/api/vlabs/validate.go

```diff
@@ -357,6 +357,11 @@ func (a *Properties) Validate(isUpdate bool) error {
 			switch a.OrchestratorProfile.OrchestratorType {
 			case DCOS:
 			case Kubernetes:
+				for _, l := range agentPoolProfile.CustomNodeLabels {
```

Comment:
To confirm is this the label that was added a couple of weeks ago what the label can be the name of resource group? If yes, I would set the limit to 90 character since that's ARM resource group length limit.

Reply:
@amanohar no this is for custom node labels

Reply:
Also https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set  " Valid label values must be 63 characters or less and must be empty or begin and end with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between."

Reply:
thanks!